x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -292,6 +284,26 @@ class PackagePolicyService implements PackagePolicyServiceInterface {
     };
   }
 
+  public async findAllForPolicy(
```

Comment:
nit: the name of this function could be improved e.g. `findAllPackagePoliciesForAgentPolicy`